code-challenges.js

```diff
@@ -58,12 +110,27 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+const indexOfValue = (value) => {
+    return numberOfConnections.lastIndexOf(value)
+}
+console.log(indexOfValue(givenValue1))
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+// input: sanDiegoSummerTemperatures, sanDiegoWinterTemperatures
+// output: array from largest to smallest
+// - make a function using the array as a parameter
+// - sort through the array using .sort() funcition
+// - console log the array from largest to smallest
 
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
 
 const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+const arraySort = (sdTemperatures) => {
+    return sdTemperatures.sort((a,b) => b-a)
+}
```

Comment:
Would be good to add how the callback function in the `.sort` method sorts the numbers in descending order.